db/migrate/20190424020531_add_user_id_to_branches.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserIdToBranches < ActiveRecord::Migration[5.2]
+  def change
+    add_column :branches, :user_id, :bigint
```

Comment:
Close. Line 3 should be:
```ruby
add_reference :branches, :user, foreign_key: true, index:true
```

You can run `rails db:rollback` to roll back the database by one step, make the change in this file, then run `rails db:migrate`.